spec/integration/kibana/spec.yml

```diff
@@ -1,121 +1,133 @@
-    - description: Folder containing Kibana tags
-      type: folder
-      name: tag
-      required: false
-      contents:
-      - description: A dashboard tag file
-        type: file
-        contentMediaType: "application/json"
-        pattern: '^{PACKAGE_NAME}-.+\.json$'
-    - description: Folder containing Osquery pack assets
-      type: folder
-      name: osquery_pack_asset
-      required: false
-      contents:
-      - description: An osquery pack asset file
-        type: file
-        contentMediaType: "application/json"
-        pattern: '^{PACKAGE_NAME}-.+\.json$'
-    - description: Folder containing Osquery saved queries
-      type: folder
-      name: osquery_saved_query
-      required: false
-      contents:
-      - description: An osquery saved query file
+  - description: Folder containing Kibana index pattern assets
+    type: folder
+    name: "index_pattern"
+    required: false
+    contents:
+    - description: An index pattern asset file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^.+\.json$'
+  - description: Folder containing rules
+    type: folder
+    name: "security_rule"
+    required: false
+    contents:
+    - description: An individual rule file for the detection engine
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^.+\.json$'
+  - description: Folder containing CSP rule templates
+    type: folder
+    name: "csp_rule_template"
+    required: false
+    contents:
+    - description: An individual CSP rule template file for the cloud security posture management solution
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^.+\.json$'
+  - description: Folder containing ML module assets
+    type: folder
+    name: ml_module
+    required: false
+    contents:
+      - description: An ML module asset file
         type: file
         contentMediaType: "application/json"
         pattern: '^{PACKAGE_NAME}-.+\.json$'
+  - description: Folder containing Kibana tags
+    type: folder
+    name: tag
+    required: false
+    contents:
+    - description: A dashboard tag file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
+  - description: Folder containing Osquery pack assets
+    type: folder
+    name: osquery_pack_asset
+    required: false
+    contents:
+    - description: An osquery pack asset file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
+  - description: Folder containing Osquery saved queries
+    type: folder
+    name: osquery_saved_query
+    required: false
+    contents:
+    - description: An osquery saved query file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
+  - description: File containing saved object tag definitions for assets
+    type: file
+    contentMediaType: "application/x-yaml"
+    name: "tags.yml"
```

Comment:
The original request was to add tags to the main `manifest.yml`, do we recommend to add to a separate file?

Reply:
> @elastic/ecosystem WDYT about creating a new file for these tag definitions ?

I'd prefer to have these definitions separated from the manifest file, yes.

Reply:
Let's do that then 👍